src/main/server.ts

```diff
@@ -1,27 +1,12 @@
 #!/usr/bin/env node
-import * as logging from "@hmcts/nodejs-logging";
-import * as fs from "fs";
-import * as https from "https";
-import * as path from "path";
+import { Logger } from "@hmcts/nodejs-logging";
 import { app } from "./app";
```

Comment:
why Logger is in upper case

Reply:
It's the named export, see: https://github.com/hmcts/nodejs-logging/blob/master/index.js#L4

Reply:
I was on the train when typing. Thx for link